contents/blog/why-we-made-surveys.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2023-10-19
+title: Why we've launched PostHog user surveys
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - joe-martin
+featuredImage: ../images/blog/posthog-survey.png
+featuredImageType: full
+category: Inside PostHog
+---
+
+Today, we’ve announced user surveys are out of beta and in general release, complete with [new pricing](/pricing). Short version? Pricing is usage-based, with a free tier for small projects and an improved offer for everyone who participated in the beta. 
+
+Launches like this are always exciting for us, but this one is _especially_ notable because it firmly establishes PostHog as an all-in-one tool for building better products. 
+
+Why is this important? Because spreading everything out across multiple platforms _sucks_. Replays in HotJar, feature flags in LaunchDarkly, analytics in Amplitude (but not your SQL insights, those go somewhere else), and experiments in Google Optimize? That only lays a foundation for frustration, incompatibility, and inconsistency.
+
+It’s also a pain having to log in and out of new platforms all the time. 
+
+Imagine instead if you could create surveys triggered by the same events you track in your analytics, or if you could trace a survey response back to a session replay in a single platform. Imagine if you could test changes with an A/B experiment and launch surveys alongside them to collect feedback, then turn that into a feature flag and roll it out without ever switching tools.
+
+Imagine if you had your qualitative and quantitative data side-by-side, along with everything you need to dissect it and act on your findings. 
+
+That’s the vision we have for PostHog — an all-in-one platform where you can research users, observe usage, analyze data, test ideas, deploy changes, and measure impact consistently, and easily. No more tool juggling, compatibility wrestling, unclear pricing, or botched integrations; just a single, open-source platform that does it all.
```

Comment:
```suggestion
Imagine creating surveys triggered by the same events you track in your analytics

Imagine tracing a survey response back to session replay in just a few clicks.

Imagine trialling a new feature as an A/B test, collecting feedback on it with surveys, and finally shipping the new feature to everyone, without ever switching tools.

Imagine your qualitative and quantitative data in one place, connected to the tools you need dissect and act on it.

That’s the vision we have for PostHog – an all-in-one platform where you can research users, observe usage, analyze data, test ideas, deploy changes, and measure impact consistently, and easily. 

No more tool juggling, compatibility wrestling, unclear pricing, or botched integrations – just a single, open-source platform that does it all.
```

Reply:
I think it's easier to parse + more impactful if we're liberal with the line breaks.